index.html

```diff
@@ -25,7 +25,165 @@
         <h1>I'm Andrea.<br>Glad to see you!</h1>
         <h2>I'm Andrea.<br>Glad to see you!</h2>
         <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
-          <div class="enlaces">
+        <div class="enlaces">
+          <p>Let's conect</p>
+          <ul>
+            <li>
+              <a href=""><img src="Img/twitter.svg" alt=""></a>
+            </li>
+            <li>
+              <a href=""><img src="Img/Linkedin icon.svg" alt=""></a>
+            </li>
+            <li>
+              <a href=""><img src="Img/medium.svg" alt=""></a>
+            </li>
+            <li>
+              <a href=""><img src="Img/github.svg" alt=""></a>
+            </li>
+            <li>
+              <a href=""><img src="Img/angellist.svg" alt=""></a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </section>
+    <section class="works">
+      <div class="card1 cg">
+        <img src="Img/Snapshoot Portfolio.png" alt="">  
+        <div class="c1desc">
+          <p class="c1dp">Tonic</p>
+          <ul>
+            <li class="li1">
+              CANOPY
+            </li>
+            <li>
+              <img src="Img/Counter.png" alt="">
+            </li>
+            <li class="lisec">
+              Back End Dev
+            </li>
+            <li>
+              <img src="Img/Counter.png" alt="">
+            </li>
+            <li class="lisec">
+              2015
+            </li>
+          </ul>
+        </div>
+        <p class="ca1p">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <div class="tags">
+          <ul>
+            <li class="norm">HTML</li>
+            <li class="norm">CSS</li>
+            <li class="special">JavaScript</li>
+          </ul>
+        </div>
+        <button class="efecbot">See project</button>
```

Comment:
- [ ] Great job adding the hover, & active styles :smile:, but in order to match the Figma design the text color should change to white when hovered or pressed. Kindly, include these styles in your button.
![Screenshot from 2023-04-29 04-49-30](https://user-images.githubusercontent.com/86069740/235278062-da07bf04-4b8b-4b9d-b41b-8bc9728e715c.png)